### ISSUE TITLE
Missing HEADER entry

### ISSUE THREAD

**Issue by JohnPhoto** [evenchange4/react-intl-po#54]
I would like to use this library together with [zanata-js](https://www.npmjs.com/package/zanata-js).
The problem I am struggling with is that zanata assumes that the header entry exists which is "defined" here: [https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html](https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html)

Which boils down to something like this:
```
# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#: src/link/to/my/src:67
msgid ""
msgstr ""
"Project-Id-Version: xxx\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-07-06 12:31-0500\n"
"PO-Revision-Date: 2011-05-27 22:37+0100\n"
"Last-Translator: My NAME<me@mydomain.com>\n"
"Language-Team: TEAM <team@TTEAM.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
```

In my research it looks like the most vital parts is the empty `msgid ""` and `msgstr ""` which react-intl-po does not add at the beginning of the .pot-file.

It would be nice to have a way with this project to inject the head entry.

**Comment by evenchange4** [comment]
Sorry, I do not have much time for this feature recently. Any PR is welcome. 😀

**Comment by Guibod** [comment]
If you want to run my version until the PR is accepted:
npm i https://github.com/Guibod/react-intl-po.git#installable

**Comment by evenchange4** [comment]
Thanks! Released `v1.2.0` with your header feature.
